openapi-cli/src/test/resources/generators/client/ballerina/uber_openapi.bal

```diff
@@ -0,0 +1,92 @@
+import  ballerina/http;
+import  ballerina/url;
+import  ballerina/lang.'string;
+
+type ProductArr Product[];
+
+type PriceEstimateArr PriceEstimate[];
+
+public client class Client {
+    public http:Client clientEp;
+    public isolated function init(string serviceUrl = "https://api.uber.com/v1", http:ClientConfiguration  httpClientConfig =  {}) returns error? {
+        http:Client httpEp = check new (serviceUrl, httpClientConfig);
+        self.clientEp = httpEp;
+    }
+    remote isolated function  products(decimal latitude, decimal longitude) returns ProductArr|error {
```

Comment:
There are 2 spaces between the `function` keyword and function name. Check for other similar places.